libs/dmlf/examples/mnist_distributed_learning/main.cpp

```diff
@@ -38,6 +38,27 @@ using TensorType       = fetch::math::Tensor<DataType>;
 using VectorTensorType = std::vector<TensorType>;
 using SizeType         = fetch::math::SizeType;
 
+/*  JSON Config:
```

Comment:
at first glance i thought this was left in code. This is intended to be an example json config that should work well for this example, right?
Can you reword to example JSON Config file to be explicit please?